### ISSUE TITLE
"Unsafe proxy detected"

### ISSUE THREAD

**Issue by johnnyfronny** [CybriaTech/CybriaGG#8]
![Screenshot 2024-03-12 10 22 30 AM](https://github.com/CybriaTech/CybriaGG/assets/162928705/06c28813-fde3-43e3-8916-1556016d5427)

**Comment by ghost** [comment]
I have been made aware of this issue, for me. It seems like Roblox does load at some points, probably due to the fact the Proxy Detection is delayed, use that glitch for now. While I try to add features to it to prevent Now.GG's proxy detection

**Comment by ghost** [comment]
Hm, it seems like CybriaGG isnt the only one affected, it's Shuttle's and even Asteroid's too. Now.GG must've updated the proxy detection

**Comment by johnnyfronny** [comment]
Yeah although rarely when I reload it or first go on some of the sites you have the proxy is delayed and it lets me log on before it can prevent it. But it definitely wasn't doing that before so they added some type of security measures.

**Comment by 1coolmatt8083** [comment]
WHYYYYYYYYYYYYYYY!!!!!!!!!!!!??????
